.devcontainer/Dockerfile

```
@@ -1,11 +1,9 @@
 FROM ubuntu:focal
 
-ENV DEBIAN_FRONTEND=noninteractive
-
-RUN apt-get update -qq
-RUN apt-get upgrade -qq
-RUN apt-get install python3 -qq
-RUN apt-get install python3-pip -qq
+RUN DEBIAN_FRONTEND=noninteractive \
```

```suggestion
RUN export DEBIAN_FRONTEND=noninteractive \
```

Maybe try that if it is still failing